types/assert-plus/tslint.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "@definitelytyped/dtslint/dt.json",
     "rules": {
+        "ban-types": false,
```

Comment:
Why is this being changed? What type is banned that is being used here?

Reply:
the `Function` type is banned